test/integration/compression_test.exs

```diff
@@ -28,7 +28,7 @@ defmodule CompressionTest do
     start_options: start_options
   } do
     assert {:ok, compressed_conn} =
-             Xandra.start_link(start_options ++ [compressor: Snappy, idle_timeout: 200])
+             Xandra.start_link(start_options ++ [compressor: Snappy, idle_interval: 200])
```

Comment:
`idle_timeout` does not appear to be a valid option! I believe this should have been `idle_interval`, which is a valid option. I thus changed this over.